support/Pipelines/Bbh/Inspiral.py

```diff
@@ -3,23 +3,161 @@
 
 import logging
 import subprocess
+from glob import glob
 from pathlib import Path
 from typing import Optional, Union
 
 import click
+import numpy as np
 import yaml
 from rich.pretty import pretty_repr
 
+import spectre.IO.H5 as spectre_h5
+from spectre.Pipelines.Bbh.FindHorizon import find_horizon
+from spectre.SphericalHarmonics import Strahlkorper
 from spectre.support.Schedule import schedule, scheduler_options
+from spectre.Visualization.ReadH5 import select_observation
 
 logger = logging.getLogger(__name__)
 
 INSPIRAL_INPUT_FILE_TEMPLATE = Path(__file__).parent / "Inspiral.yaml"
 
 
+def _find_horizon(
```

Comment:
I moved this to a `postprocess_id` step that runs automatically after ID generation: #6082. Then the masses and spins are just stored alongside the ID in `BbhReductions.h5:Ah{A,B}.dat` and you can just read them here. That's better, right?

Reply:
Naively, it does seem better. But we should probably discuss more offline about how this post-process step will interact with ID root find, ecc reduction, PB&J (which we'll need to do more planning about), and whether we want yet another step in the pipeline (too few and we don't have enough flexibility; too many and it gets confusing)